specs/ExtendedNewWindowRequested.md

```diff
@@ -0,0 +1,115 @@
+# Background
+WebView2 raises the `NewWindowRequested` event when a new window is opened. The WebView2 team has been asked to include the name of the new window 
```

Comment:
When a new window is _opened_, or when the control wants to open a window?

Reply:
Yes we can make this sentence clearer
```suggestion
WebView2 raises the `NewWindowRequested` event when web content in the WebView2 tries to open a new window. The WebView2 team has been asked to include the name of the new window 
```

Reply:
Made clearer now.